kde-plasma/discover/discover-5.27.49.9999.ebuild

```diff
@@ -67,11 +69,16 @@ src_prepare() {
 
 src_configure() {
 	local mycmakeargs=(
+		# TODO: Port PackageKit's portage back-end to python3 and
+		# move from layman to eselect-repository
 		-DCMAKE_DISABLE_FIND_PACKAGE_packagekitqt5=ON
-		-DCMAKE_DISABLE_FIND_PACKAGE_Snapd=ON
-		-DWITH_KCM=OFF
+		-DWITH_KCM=ON
```

Comment:
Why do we enable that?

https://invent.kde.org/plasma/discover/-/merge_requests/142

Reply:
> No objections, though reasons need to be given for enabling the KCM.

I had enabled this out of curiosity, I thought it might allow for automatically updating snaps and flatpaks. But I have been running with this enabled for a while and this appears to not be the case (I have one flatpak installed and it is not updating automatically). I have disabled this again and added a comment indicating why it is disabled.